tasks/ssh.yml

```diff
@@ -21,10 +21,13 @@
   register: __kdump_ssh_user_info
   delegate_to: "{{ kdump_ssh_server }}"
 
+- name: Set ssh file path
```

Comment:
Can you set it in the below task or this doesn't work?

Reply:
> Can you set it in the below task or this doesn't work?

It doesn't work.  `set_fact` variable evaluation happens immediately - so you get a `__kdump_ssh_path is undefined` error when evaluating `__kdump_authorized_keys_path` in the same `set_fact` - not sure why it works differently than `vars` or other variable settings.